### DISCUSSION TITLE
Remote play away from local network

### DISCUSSION THREAD

**Discussion by Jos552** [unknownskl/xbox-xcloud-client#485] [Q&A]
Is there any way to get remote play to work away from home network on the steam deck? I have forwarded the appropriate ports but still not working. I was able to get it to work on PS5 with chiaki doing port forwarding. Thanks!

**Comment by bmcgreer** [comment]
FWIW, I got it to work with my Macbook Pro by setting up the VPN on my router, installing a VPN client on my laptop, and using that to route just the traffic destined for my home network through the VPN.

**Reply by Ouden-Lin** [reply]
I also use a VPN. But it depends a lot on the connection quality.

**Comment by NotTheVacuum** [comment]
Edit: pardon the error, remote play does work over the internet for Series consoles natively

**Reply by rackerby** [reply]
And Xbox One, One X, One S.

https://support.xbox.com/en-US/help/games-apps/game-setup-and-play/how-to-set-up-remote-play

System requirements: Xbox One or later consoles 

But again, this is for the official Xbox app.

**Reply by NotTheVacuum** [reply]
Yeah, clearly I was confused... I never used the Xbox app for remote on the Xbox One generation, always used another 3rd party app that also implemented a controller driver.

**Comment by rackerby** [comment]
The official Xbox app for iOS and Windows allows remote play outside the local network.

**Comment by NotTheVacuum** [comment]
Greenlight behaves interestingly when trying to stream over internet. When I’m using the local network, sometimes I get the error message that it’s unable to start streaming for a moment after the Xbox starts up. This is normal, and streaming works a few moments later. I can reproduce these errors over the internet shortly after startup, however after they subside, I get the loading screen indefinitely. 

So it seems to fail in a similar fashion shortly after startup, but even when that error subsides, it doesn’t connect. Implies there’s some level of connectivity but it just doesn’t quite work.

**Reply by rackerby** [reply]
The mobile or Windows Xbox app does work for you over the Internet, right?

**Reply by NotTheVacuum** [reply]
Yes, it does

**Comment by ahkdigital** [comment]
Also interested in knowing if this works for anyone (outside of the VPN route) - from my experience on Beta 8, it definitely turns the console on and even gets to the point of saying it's connecting but the actual video stream never starts. So, some sort of communication is definitely happening.

**Comment by rackerby** [comment]
Huh.  You are correct.
